physionet-django/events/views.py

```diff
@@ -101,8 +142,8 @@ def event_add_participant(request, event_slug):
 
     if request.method == 'POST':
         if request.POST.get('confirm_event') == 'confirm':
-            event.enroll_user(user)
-            messages.success(request, "You have been enrolled")
+            event.join_waitlist(user=user, comment='Automatic')
```

Comment:
What is the purpose of the "Automatic" comment here?

Reply:
Because the waitlist method needs us to pass a comment, i thought 'Automatic' would make sense(as in the participant was added automatically to waitlist)

Maybe something like "Approval needed as per Event Settings" would be better?

Reply:
It doesn't feel right that we are adding a comment simply because the model expects one. In this case, maybe the comment should be optional.

It makes me realize that we should be clear of the purpose of this comment (both for ourselves, and to the host). Is the comment meant to be public (i.e. sent to the applicant) or is it a "note to self"?

Reply:
Noted. for optional comment, its already optional by default on modal, i think i could just edit the `join_waitlist` method with a if statement(or pass an empty comment on the view)

I am not sure if the comment is meant to be public(but do we want to make it public). it might be helpful for the participants to know why they were rejected.

Also i had assumed that we would be sending the comment in the e-mail to the participant on the rejection email(in case we decide to add the email)

What do you think @tompollard ?

Reply:
> Also i had assumed that we would be sending the comment in the e-mail to the participant on the rejection email(in case we decide to add the email)

 I think it makes sense for the host to give an explanation to the applicant. Perhaps we should rename this something like "comment_to_applicant" in this case, to make this clear?

Reply:
that sounds good. i will make the changes

Thank you